src/DFAMin.hs

```diff
@@ -153,9 +157,21 @@ groupEquivStates DFA { dfa_states = statemap }
     go r (a:q) = uncurry go $ List.foldl' go0 (a:r,q) xs
       where
         xs :: [EquivalenceClass]
-        xs = filter (not . IS.null)
-           . map (\m -> IS.unions [IM.findWithDefault IS.empty s m | s <- IS.toList a])
-           $ IM.elems bigmap
+#if MIN_VERSION_containers(0, 6, 0)
```

Comment:
Can this just be around the line that varies between the two?

Reply:
Yeah, CPP works in string level

Reply:
Yeah I would prefer that.

Reply:
https://github.com/simonmar/alex/pull/176/commits/9f399ae5fb0ce08e67cd903c0efa30ec5002c11e
Is this right what did you wanted?

Reply:
Yup!